src/it/resources/com/google/checkstyle/test/chapter4formatting/rule487modifiers/InputModifierOrder.java

```diff
@@ -226,34 +233,42 @@ abstract class AbsInner {
     // violation above ''public' modifier out of order with the JLS suggestions.'
 
     @MyAnnotation2 public final synchronized strictfp void fooMethod() {}
+
     ;
```

Comment:
our config was complaining about `;` placement

it is weird that we needed to add a blank line between `;` and it's method. This might be a false positive in our config. Will look into this.

Reply:
What check is complaining?
I suggest to move this case to input where this Checks is primary focus.
And in this input remove all such weird ; , our focus here is order of modifiers

Reply:
It is completely ok to update in PR not target Input to move there edge cases

Reply:
> What check is complaining?

[EmptyLineSeparator](https://checkstyle.sourceforge.io/checks/whitespace/emptylineseparator.html), violation was something like:

`54:3: ';' should be separated from previous line.`

> I suggest to move this case to input where this Checks is primary focus.

[3 Source file structure](https://checkstyle.sourceforge.io/styleguides/google-java-style-20180523/javaguide.html#s3-source-file-structure) uses this check but I'm not sure if we should give violations for such code or not. This maybe false positive from our side. I will check this soon.

> And in this input remove all such weird ; , our focus here is order of modifiers

I have removed such `;`, they were unnecessary but I will look into the violation that it was causing, if it would be a false positive then I will open an issue for it